composer.json

```diff
@@ -27,7 +27,7 @@
         }
     ],
     "require": {
-        "php": "8.0 - 8.4",
+        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

Comment:
Doesn't previous format work well?

Reply:
I copied this from other repos; I can make it as "8.1 - 8.4". What is the preferred format?